[2023-08-31 13:22:06,499] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2021-09-20T00:00:00+00:00 [queued]>
[2023-08-31 13:22:06,573] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2021-09-20T00:00:00+00:00 [queued]>
[2023-08-31 13:22:06,585] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-08-31 13:22:06,586] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2023-08-31 13:22:06,586] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-08-31 13:22:06,651] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2021-09-20T00:00:00+00:00
[2023-08-31 13:22:06,659] {standard_task_runner.py:52} INFO - Started process 2689 to run task
[2023-08-31 13:22:06,675] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v5', 'second_task', '2021-09-20T00:00:00+00:00', '--job-id', '163', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp9b7aj2lc', '--error-file', '/tmp/tmp9l39ql6i']
[2023-08-31 13:22:06,683] {standard_task_runner.py:77} INFO - Job 163: Subtask second_task
[2023-08-31 13:22:06,807] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v5.second_task 2021-09-20T00:00:00+00:00 [running]> on host e7718740c1e6
[2023-08-31 13:22:06,989] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=devendra
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-09-20T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-20T00:00:00+00:00
[2023-08-31 13:22:06,990] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2023-08-31 13:22:06,992] {bash.py:158} INFO - Running command: echo hey, I am task2 and will be running after task1!
[2023-08-31 13:22:07,020] {bash.py:169} INFO - Output:
[2023-08-31 13:22:07,100] {bash.py:173} INFO - hey, I am task2 and will be running after task1!
[2023-08-31 13:22:07,120] {bash.py:177} INFO - Command exited with return code 0
[2023-08-31 13:22:07,192] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210920T000000, start_date=20230831T132206, end_date=20230831T132207
[2023-08-31 13:22:07,261] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-31 13:22:07,295] {local_task_job.py:146} INFO - Task exited with return code 0
